process/sync/basicForkDetector.go

```diff
@@ -362,14 +368,26 @@ func (bfd *basicForkDetector) CheckFork() (bool, uint64, []byte) {
 				continue
 			}
 			if hdrInfos[i].state == process.BHNotarized {
-				lowestRoundInForkNonce = 0
-				forkHeaderHash = hdrInfos[i].hash
+				if lowestRoundInForkNonce > 0 {
```

Comment:
Might refactor this multi-level if instructions

Reply:
Done